mlflow/metrics/genai/genai_metric.py

```diff
@@ -172,6 +172,8 @@ def make_genai_metric(
 
     if aggregations is None:
         aggregations = ["mean", "variance", "p90"]
+    if grading_context_columns is None:
+        grading_context_columns = []
```

Comment:
Can we add this to directly the function signature?

So users know which sensible defaults we have taken? 

```
def make_genai_metric(
    name: str,
    definition: str,
    grading_prompt: str,
    examples: Optional[List[EvaluationExample]] = None,
    version: Optional[str] = _get_latest_metric_version(),
    model: Optional[str] = "openai:/gpt-3.5-turbo-16k",
    grading_context_columns: Optional[List[str]] = [],
    parameters: Optional[Dict[str, Any]] = None,
    aggregations: Optional[List[str]] = ["mean", "variance", "p90"],
    greater_is_better: bool = True,
    max_workers: int = 10,
    judge_request_timeout: int = 15,
) -> EvaluationMetric:
```

Reply:
![Uploading Screenshot 2023-10-16 at 8.04.09 PM.png…]()